store/Cargo.toml

```diff
@@ -37,3 +37,7 @@ crc = { version = "1.8", optional = true }
 tempfile = "3.1"
 exocore-chain = { path = "../chain", features = ["tests-utils"] }
 exocore-core = { path = "../core", features = ["tests-utils"] }
+
+# For tests
+tokio = { version = "0.2", features = ["test-util", "macros", "blocking", "rt-core", "rt-threaded"], default-features = false }
```

Comment:
Should also be optional & enabled with `tests-utils`

Reply:
Added it to the main deps, gated by `tests-utils`, but it needs to remain in the dev deps for tests to run